mmv1/third_party/terraform/.teamcity/components/inputs/services_beta.kt

```diff
@@ -656,6 +656,11 @@ var ServicesListBeta = mapOf(
         "displayName" to "Resourcemanager",
         "path" to "./google-beta/services/resourcemanager"
     ),
+    "resourcemanager3" to mapOf(
```

Comment:
Also add this to services_ga.kt

It needs to be present in both even though the resource is only in beta